### ISSUE TITLE
locale-config not working with v-model

### ISSUE THREAD

**Issue by DevIsShortForDevil** [talkhabi/vue-persian-datetime-picker#116]
Hello
 locale-config as it shows in the document doesn't work properly alongside v-model and format attributes and display-formats often get mixed up and show the wrong format. I tested it alot and I feel when I use "fa,en" in locale it's more likely to show the jalaali format and when it's on "en,fa" it's the other way around.

```
<date-picker
       date-picker
        v-model="date"
        format="YYYY/MM/DD"
        locale="fa,en"
        :locale-config="{
            fa: {
             displayFormat: 'jYYYY/jMM/jDD',
             lang: { label: 'شمسی' }
            },
            en: {
                displayFormat: 'YYYY/MM/DD',
                lang: { label: 'Gregorian' }
           }
       }"
/>
```

is there a solution to this problem ? am I making a mistake somewhere or is this a bug ?

**Comment by talkhabi** [comment]
Hi @DevIsShortForDevil 
You are right, this seems to be a bug!

**Comment by talkhabi** [comment]
Thank you for reporting this issue,
This problem was fixed in version `2.1.6`.
